packages/ec-ga-cli/src/main.rs

```diff
@@ -69,7 +72,7 @@ fn main() -> Result<()> {
     };
 
     let individual_generator = ec::IndividualGenerator {
-        scorer,
+        scorer: FnScorer(scorer),
```

Comment:
Turns out that this doesn't work (at least not immediately) because there's another use of `scorer` down below (in `make_new_individual` that fails if we turn `scorer` into an `FnScorer`. I think the right answer there is actually to change `.wrap()` so that it takes an `FnScorer`. We should look at that in tomorrow's stream.